src/main/kotlin/no/nav/dagpenger/rapportering/repository/JournalfoeringRepositoryPostgres.kt

```diff
@@ -29,55 +29,37 @@ class JournalfoeringRepositoryPostgres(
         }
     }
 
-    override fun lagreJournalpostMidlertidig(journalpost: Journalpost) {
+    override fun lagreDataMidlertidig(midlertidigLagretData: MidlertidigLagretData) {
```

Comment:
Skulle vi lagt til action timer på disse funksjonene? 
Det ville i såfall sett slik ut:
```suggestion
    override fun lagreDataMidlertidig(midlertidigLagretData: MidlertidigLagretData) =
        actionTimer.timedAction("db-lagreDataMidlertidig") {
```

Reply:
Done